semester_1/kr1.2/task1/main.c

```diff
@@ -0,0 +1,54 @@
+#include <stdio.h>
+#include <stdbool.h>
+#include <locale.h>
+
+#include "compareBinaryNumbers/compareBinaryNumbers.h"
+#include "tests/test.h"
+
+#define PROGRAM_FINISHED_CORRECTLY 0
+#define PROGRAM_FAILED_TESTS 1
+
+static void printBinaryNumber(const bool *const number, const size_t len)
+{
+    for (size_t i = 0; i < len; ++i)
+    {
+        printf("%d", number[i]);
+    }
+    printf("\n");
+}
+
+int main(void)
+{
+    setlocale(LC_ALL, "Russian");
+
+    if (!test())
+    {
+        printf("Программа сейчас не работает\n");
```

Comment:
```suggestion
        printf("Уважаемый, вы что, не видите, у нас обед!\n");
```
![image](https://github.com/kamendov-maxim/university_homeworks/assets/706768/e349855d-32bf-41dd-92fa-8030bc5ba0e0)